CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This release includes:
 
 Compared to `2.25.1` this release adds:
 * Feature - Add `auto_create_stream ` option [cloudwatch:257](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/pull/257)
+* Feature - Add Apache Arrow support in S3 [s3:3184](https://github.com/fluent/fluent-bit/pull/3184)
```

Comment:
let's make it clear, since the link is to the old FB PR, let's say we enabled it at compile/build time starting in this release